pkg/3scale/amp/component/backend.go

```diff
@@ -14,6 +14,11 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	BackendListenerName = "backend-listener"
+	BackendWorkerName   = "backend-worker"
```

Comment:
I'm curious why has this not been done for backend-cron?

Reply:
Backend cron DC name is not used in upgrade path.

Should be done? I think so. Required, not at all.

Reply:
I agree. I was just curios about why was not done :+1: